src/cli/cli_app.js

```diff
@@ -407,6 +416,14 @@ class CLI {
       await n.stop();
       process.exit();
     });
+
+    if (this._autoInit) {
```

Comment:
Thanks, I've moved the code into a new function.
It can not be moved into the constructor because the `_initEnvironment` should be `await`ed before doing and that isn't possible in the constructor.